### ISSUE TITLE
OpenAPIParser.readLocation fails when fetching valid Swagger 2.0 resource with AuthorizationValues provided

### ISSUE THREAD

**Issue by charlesyu108** [swagger-api/swagger-parser#785]
I have a simple server that serves a Swagger doc when a specified header and value are provided. When I try to load the resource via
```
Swagger parseResult = new OpenAPIParser().readLocation(uri, docAuths, null);
```
where docAuths are my AuthorizationValues, the parser will fail with the message "attribute openapi is missing" if my server provides a 2.0 resource but not a 3.0 resource (the 3.0 resource I provide is the converted version of the 2.0 one using swagger-parser).

I believe that the fetched resource isn't being run through the converter first when Authorization is provided.

**Comment by gracekarina** [comment]
Hi @charlesyu108, can you please provide a PR with a test so we can work on it. Thanks!

**Comment by ymohdriz** [comment]
Hi @charlesyu108 

There was an issue in auth conversion code, it's fixed now in PR #789 

Thanks,
Mohammed

**Comment by gracekarina** [comment]
Fixed by #789
